lib/fizzy/execute.cpp

```diff
@@ -192,7 +193,19 @@ Instance instantiate(const Module& module)
         }
     }
 
-    Instance instance = {module, std::move(memory), memory_max, std::move(globals)};
+    std::vector<TypeIdx> imported_function_types;
```

Comment:
Here we collect all types of the imported functions from `importsec` vector, which contains everything (functions, tables, memories, globals).

We could do it in parser, but then `importsec` would not be `vector<Import>` anymore.